src/config/fenix.js

```diff
@@ -0,0 +1,127 @@
+import { transformAPIResponse } from '../utils/transform-data';
+import { extractBucketMetadata } from './shared';
+import { getProbeData } from '../state/api';
+import { validate, noResponse } from '../utils/data-validation';
+
+export default {
+  sampleRate: 1,
+  dimensions: {
+    os: {
+      title: 'OS',
+      key: 'os',
+      values: [{ key: 'Android', label: 'Android' }],
+    },
+    channel: {
+      title: 'Channel',
+      key: 'channel',
+      values: [
+        { key: 'fenixProduction', label: 'Firefox Preview' },
+        { key: 'release', label: 'Fenix' },
+      ],
+    },
+    pingType: {
+      title: 'Ping Type',
+      key: 'ping_type',
+      values: [
+        { key: '*', label: 'All' },
+        { key: 'metrics', label: 'Metrics' },
+        { key: 'baseline', label: 'Baseline' },
+      ],
+    },
+    aggregationLevel: {
+      title: 'Aggregation Level',
+      key: 'aggregationLevel',
+      values: [
+        { key: 'build_id', label: 'Build ID' },
+        { key: 'version', label: 'Major Version' },
+      ],
+    },
```

Comment:
Assuming we keep the approach from my PR, we'll need to add `defaultValue` properties for each of these.